python/mxnet/test_utils.py

```diff
@@ -1441,6 +1441,67 @@ def read_data(label_url, image_url):
     return {'train_data':train_img, 'train_label':train_lbl,
             'test_data':test_img, 'test_label':test_lbl}
 
+# download mnist.pkl.gz
```

Comment:
Instead of as comments, this might be better as a docstring. Same for other functions. Also explicitly call out that this returns none.

Reply:
Yeah, sorry. Had moved these existing functions to test_utils, but hadn't updated the comments. Added docstrings now

Reply:
Is there a particular way for saying this returns nothing?
```
Returns
--------
None
```
?

Reply:
It would be nice to refactor the test utils to use gluon.
However, in the examples - especially gluon ones - let's not hide the process in a test utility. It is succinct enough as is, and it is better to show people how to directly interact with the gluon interfaces.